Scripts/Runtime/Selection.cs

```diff
@@ -1,14 +1,65 @@
 ﻿#if !UNITY_EDITOR
+using System.Linq;
 using UnityEngine;
 
 // Not fully implemented yet; Exists only to allow compilation
 public static class Selection
 {
-    public static GameObject activeGameObject { get; set; }
-    public static int activeInstanceID { get; set; }
-    public static Transform activeTransform { get; set; }
-    public static GameObject[] gameObjects { get; set; }
-    public static Object[] objects { get; set; }
-    public static Transform[] transforms { get; set; }
+    public static Object activeObject
+    {
+        get {return s_Objects != null && s_Objects.Length > 0 ? s_Objects[0] : null; }
+        set { s_Objects = value ? new[] { value } : null; }
+    }
+
+    public static GameObject activeGameObject
+    {
+        get {return s_Objects != null && s_Objects.Length > 0 ? s_Objects[0] as GameObject : null; }
+        set { s_Objects = value ? new[] { value } : null; }
```

Comment:
Can s_Objects ever hold components? If so, we might want to handle that case and bubble up the owning game object

Reply:
Yeah, this was a quick hack. Seems to work for our needs at the moment though.

Reply:
@amirebrahimi, what more needs to be done here? I was able to select and manipulate an object in play mode.

Reply:
I think this is good enough for now. It's just a shim that can be replaced maybe at some point if runtime support is ever fully well...supported.